### PR TITLE
🎉 icons v2.0.0 

### PR DESCRIPTION
**Pull request by domhorvathtw**
## ❓ Context
Icons v2 official release after many beta releases. For more details, check the commits and the linked PRs. 

## ✅ Checklist

- [x] The package version is bumped according to README in `package.json`, `package-lock.json`, and `CHANGELOG.md`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -108,37 +71,23 @@ workflows:
...
-              only: master
-      - deploy-to-aws:
-          requires:
-            - test
-            - build-package
+            - build
           filters:
             branches:
               only: master
-      - demo:
```
Comment (oliverviljamaa) [inline .circleci/config.yml:137]: Are we working on adding a demo page?

File: README.md [hunk 2]
```diff
@@ -1,135 +1,156 @@
-# TransferWise icons
+# TransferWise SVG Icons
```
Comment (oliverviljamaa) [inline README.md:1]: Why is this change necessary?

File: .circleci/config.yml (hunk 1 above)
Comment (domhorvathtw) [reply .circleci/config.yml:137]: yes, I'm working on it, trying to finish it before so I can demo it on the frontend guild meeting

File: README.md (hunk 2 above)
Comment (domhorvathtw) [reply README.md:1]: I don't think it's necessary, @dozortsev I think it was you, if I remember correctly

Comment (domhorvathtw) [reply README.md:1]: I'm happy to go with both, although from the rest of the README it's crystal clear that these are SVG icons

Comment (oliverviljamaa) [reply README.md:1]: Yup, I'd keep it short and to the point.

File: README.md [hunk 3]
```diff
@@ -1,135 +1,156 @@
...
-## Usage
+Set of SVG icons designed and built by TransferWise and distributed as React and AngularJS components.
+Each Icon is designed in two sizes: **`16`** and **`24`** pixels.
+The icon set has mostly **`outline`** icons but some of them also have **`filled`** variants.
 
-[A helpful general guide for SVG icon sprites - CSSTricks](https://css-tricks.com/svg-sprites-use-better-icon-fonts/)
+React and AngularJS icon components generated by TypeScript script files (see `/scripts` folder).
 
+#### TransferWise Icons v1 
+You can find the documentation of v1 icons library [here](https://github.com/transferwise/icons/blob/f2d4b29847b3fa9ca488dac8acd5e2b75fc74291/README.md).
```
Comment (oliverviljamaa) [inline README.md:22]: I think that can be moved down a bit, it's not as important as usage.

File: README.md [hunk 4]
```diff
@@ -1,135 +1,156 @@
...
+
+# Usage
```
Comment (oliverviljamaa) [inline README.md:24]: Let's make sure our heading hierarchy makes sense.

File: README.md [hunk 5]
```diff
@@ -1,135 +1,156 @@
...
-### Usage from a non-module-based environment
+# Styling
 
-#### JavaScript
+Each component has a general `.tw-icon` CSS class and a specific one that includes the icon's name in kebab-case, e.g `.tw-icon-card-transferwise`
 
-1. Add `node_modules/@transferwise/icons/dist/svg-icon-sprite.js` to your bundle.
-1. Use `window.svgIconSprite` from your code to access the sprite string
-1. Create components in your framework if you want to
+### Coloring Icons
```
Comment (oliverviljamaa) [inline README.md:91]:
```suggestion
### Coloring icons
```

File: README.md [hunk 6]
```diff
@@ -1,135 +1,156 @@
...
+}
+
+/* change the color of the icon on :hover */
+.parent-element:hover > .tw-icon-fast-flag {
+  color: red;
+}
 ```
 
-### Usage from server side
+### React Components
```
Comment (oliverviljamaa) [inline README.md:119]:
```suggestion
### React components
```

File: README.md [hunk 7]
```diff
@@ -1,135 +1,156 @@
...
+```jsx
+<BankIcon size={24} filled className="m-t-2" />
+```
 
-### Styles
+# Accessibility
 
-The package comes with styles (`icons.min.css`) for icon sizes you should use
-in your components for consistency.
+### React Components
```
Comment (oliverviljamaa) [inline README.md:129]:
```suggestion
### React components
```
